packages/wallet/backend/src/rapyd/schemas.ts

```diff
@@ -37,15 +37,6 @@ export const walletSchema = z.object({
   })
 })
 
-export const ratesSchema = z.object({
```

Comment:
schemas for rapyd should not be moved

Reply:
removed from here and use this definition from rates/validation